src/assets/styles/SignStyle.js

```diff
@@ -0,0 +1,116 @@
+import styled from 'styled-components';
+
+const StyledPageContainer = styled.div`
+  background-color: #ae3e3e;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+`;
+const StyledSignContent = styled.div`
+  width: 100vw;
+  max-width: 950px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding: 0 30px 0 30px;
```

Comment:
There is barely any space between the logo and the address bar. Isn't it better if the space between the subtitle and the first input was smaller or if the subtitle was removed altogether to add some space above the logo and below the link to the login page?
![image](https://user-images.githubusercontent.com/72293497/141205681-4f4f8cbd-0790-4e80-8e53-b7b208de98dc.png)

Reply:
![image](https://user-images.githubusercontent.com/84607762/141226468-9f4f7007-4324-4390-8278-c53ab2924429.png)

Reply:
Na minha tela está ótimo. Deve ser por causa dos diferentes tamanhos das telas. Confere se vc não abriu a página com zoom do browser.

Reply:
De qualquer forma vou diminuir o tamanho de tudo para ficar melhor

Reply:
Provavelmente é pelo tamanho da tela. O zoom não está ativado.